rules/mockery-to-prophecy/src/Rector/StaticCall/MockeryCloseRemoveRector.php

```diff
@@ -0,0 +1,62 @@
+<?php
+
+declare(strict_types=1);
+
+namespace Rector\MockeryToProphecy\Rector\StaticCall;
+
+use PhpParser\Node;
+use PhpParser\Node\Expr\StaticCall;
+use Rector\Core\Rector\AbstractPHPUnitRector;
+use Rector\Core\RectorDefinition\CodeSample;
+use Rector\Core\RectorDefinition\RectorDefinition;
+
+/**
+ * @see \Rector\MockeryToProphecy\Tests\Rector\StaticCall\MockeryToProphecyRector\MockeryToProphecyRectorTest
+ */
+final class MockeryCloseRemoveRector extends AbstractPHPUnitRector
+{
+    /**
+     * @return string[]
+     */
+    public function getNodeTypes(): array
+    {
+        return [StaticCall::class];
+    }
+
```

Comment:
Here should be @param with smae node as in `getNodeType()`

To make static analysis and instant refactoring work

```php
/**
 * @param StaticCall $node
 */
```